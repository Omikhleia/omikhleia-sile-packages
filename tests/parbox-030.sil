\begin{document}
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 030 = From the package documentation, alignment with rule struts

(3A) \parbox[width=37%fw, valign=top, strut=rule, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\smallskip

(3B) \parbox[width=37%fw, valign=bottom, strut=rule, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\smallskip

(3C) \parbox[width=37%fw, valign=middle, strut=rule, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\end{document}