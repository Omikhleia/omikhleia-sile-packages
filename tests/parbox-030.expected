Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	54.0945
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	32.9327
T	22 36 w=10.6445	(3A)
Mx 	43.5773
T	12 w=3.1689	())
Draw line	49.4238	42.0945	198.2268	0.5000
Draw line	49.4238	93.6945	198.2268	0.5000
Draw line	49.4238	42.0945	0.5000	52.1000
Draw line	247.6506	42.0945	0.5000	52.1000
Mx 	49.4238
T	82 81 72 w=15.1660	(one)
My 	66.0945
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	78.2510
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.9278
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	134.3331
T	86 76 87 w=10.0146	(sit)
Mx 	148.8243
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.9845
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.6213
T	86 68 w=8.4521	(sa)
Mx 	231.0734
T	71 76 83 w=13.2080	(dip)
Mx 	244.2814
T	16 w=3.3691	(-)
Mx 	49.4238
My 	78.0945
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	73.4215
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	49.4238
My 	90.0945
T	87 90 82 w=15.5322	(two)
Mx 	29.7638
My 	145.6945
T	11 w=3.1689	(()
Mx 	32.9327
T	22 37 w=10.1904	(3B)
Mx 	43.1232
T	12 w=3.1689	())
Draw line	48.9697	97.6945	198.2268	0.5000
Draw line	48.9697	149.2945	198.2268	0.5000
Draw line	48.9697	97.6945	0.5000	52.1000
Draw line	247.1965	97.6945	0.5000	52.1000
Mx 	48.9697
My 	109.6945
T	82 81 72 w=15.1660	(one)
My 	121.6945
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	77.7969
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.4737
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	133.8790
T	86 76 87 w=10.0146	(sit)
Mx 	148.3703
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.5305
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.1672
T	86 68 w=8.4521	(sa)
Mx 	230.6194
T	71 76 83 w=13.2080	(dip)
Mx 	243.8274
T	16 w=3.3691	(-)
Mx 	48.9697
My 	133.6945
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	72.9675
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	48.9697
My 	145.6945
T	87 90 82 w=15.5322	(two)
Mx 	29.7638
My 	179.0945
T	11 w=3.1689	(()
Mx 	32.9327
T	22 38 w=10.0537	(3C)
Mx 	42.9864
T	12 w=3.1689	())
Draw line	48.8330	153.2945	198.2268	0.5000
Draw line	48.8330	204.8945	198.2268	0.5000
Draw line	48.8330	153.2945	0.5000	52.1000
Draw line	247.0598	153.2945	0.5000	52.1000
Mx 	48.8330
My 	165.2945
T	82 81 72 w=15.1660	(one)
My 	177.2945
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	77.6602
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.3370
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	133.7423
T	86 76 87 w=10.0146	(sit)
Mx 	148.2336
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.3937
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.0305
T	86 68 w=8.4521	(sa)
Mx 	230.4826
T	71 76 83 w=13.2080	(dip)
Mx 	243.6907
T	16 w=3.3691	(-)
Mx 	48.8330
My 	189.2945
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	72.8308
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	48.8330
My 	201.2945
T	87 90 82 w=15.5322	(two)
End page
Finish
